types/node/index.d.ts

```diff
@@ -2542,12 +2567,17 @@ declare module "net" {
         prependOnceListener(event: "error", listener: (err: Error) => void): this;
         prependOnceListener(event: "listening", listener: () => void): this;
     }
+
+    export interface NetConnectOpts extends TcpSocketConnectOpts, IpcSocketConnectOpts, SocketConstructorOpts {
```

Comment:
This will no longer work once property `path` in `IpcSocketConnectOpts` is mandatory.